src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqExpectationsRepository.kt

```diff
@@ -93,19 +93,31 @@ class JooqExpectationsRepository(dsl: DSLContext)
                 RESPONSIBILITY_SET.MODELLING_GROUP,
                 *BURDEN_ESTIMATE_EXPECTATION.fields(),
                 SCENARIO.SCENARIO_DESCRIPTION,
-                Tables.outcomes.OUTCOME)
+                BURDEN_OUTCOME.CODE,
+                BURDEN_OUTCOME.NAME)
                 .fromJoinPath(RESPONSIBILITY, BURDEN_ESTIMATE_EXPECTATION)
                 .joinPath(RESPONSIBILITY, SCENARIO, SCENARIO_DESCRIPTION)
                 .joinPath(RESPONSIBILITY, RESPONSIBILITY_SET, TOUCHSTONE)
                 .joinPath(BURDEN_ESTIMATE_EXPECTATION, Tables.outcomes, joinType = JoinType.LEFT_OUTER_JOIN)
+                .leftJoin(BURDEN_OUTCOME)
```

Comment:
JoinPath didn't seem to work here, I think perhaps because it was trying to join on BURDEN_OUTCOME's id column rather than code column.